### ISSUE TITLE
Corrplot: no se ve bien

### ISSUE THREAD

**Issue by PaulaMorBla14** [PaulaMorBla14/TFG-Baloncesto#21]
https://github.com/PaulaMorBla14/TFG-Baloncesto/blob/9f86d010b14c2c251e5dafad4a37e119f524de77/TFG_MetodosEstadisticosBaloncesto.Rnw#L590

Como hay un total de 1200 jugadores, el corplot se ve demasiado pequeño, por lo que no es útil. Como puedo hacer para que no tenga más sentido?

**Comment by solmos** [comment]
Ya te comenté que tendría solo sentido ver dentro de cada equipo, qué jugadores tienen las correlaciones más altas de estar juntos en pista. Por darte alguna idea, quizás puedes calcular esta correlación entre jugadores de un mismo equipo y mostrar solo las más altas en el corrplot.

**Comment by PaulaMorBla14** [comment]
Cierto! se me había olvidado. Vale ahora lo miro de hacer. Gracias!
